dev/airflow-jira

```diff
@@ -134,7 +134,7 @@ def compare(target_version):
 
     for issue in issues:
         is_merged = issue.key in merges
-        print("{:<18}|{:<12}||{:<10}||{:<10}|{:<50}|{:<6}|{:<6}|{:<40}"
+        print("{:<18}|{!s:<12}||{!s:<10}||{!s:<10}|{:<50}|{:<6}|{:<6}|{:<40}"
```

Comment:
what is this line change trying to fix?

Reply:
This line of code is begging for documentation!

Reply:
@feng-tao @r39132 Updated. PTAL.